### PR TITLE
Improve labels used to describe the scoring

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

File: httpobsdashboard/templates/index.html [hunk 1]
```diff
@@ -104,17 +104,17 @@
...
+                        <span class="grade" data-content="Over the past 90 days there has been a score change of: <strong>{{ results.httpobs.delta90 }}/100</strong>" data-pass="{% if results.httpobs.delta90 > 0 %}True{% elif results.httpobs.delta90 == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
                         {{ results['httpobs']['delta90'] }}
                         </span>
                     </td>
                     <td class="text-center">
-                        <span class="grade" data-content="{{ results.httpobs.delta }}" data-pass="{% if results.httpobs.delta > 0 %}True{% elif results.httpobs.delta == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
+                        <span class="grade" data-content="Since records began there has been a score change of: <strong>{{ results.httpobs.delta }}/100</strong>" data-pass="{% if results.httpobs.delta > 0 %}True{% elif results.httpobs.delta == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
```
Comment (jonathanKingston) [inline httpobsdashboard/templates/index.html:112]: @marumari I just noticed 100 isn't the top score. Is the current top score 120?

**Comment by floatingatoll** [comment]
I think we're at 125 or 130 on phonebook once you're past SSO, and I know
we're missing Referrer policy and HSTS Preload. Might be 135-140 range or
more.

 - R.

File: httpobsdashboard/templates/index.html (hunk 1 above)
Comment (april) [reply httpobsdashboard/templates/index.html:112]: Currently the top score is 135:

https://github.com/mozilla/http-observatory/blob/master/httpobs/docs/scoring.md

Comment (april) [reply httpobsdashboard/templates/index.html:112]: You could probably just say that there has been a score change of X, not X/100.

Comment (jonathanKingston) [reply httpobsdashboard/templates/index.html:112]: @marumari I can make it a var from the python script, it seems nicer to tell users more info on hover like this. Happy to remove though too :)

**Comment by floatingatoll** [comment]
The tiniest nitpick in the world here: It would be great if the delta said
+X, -X rather than X, -X, since most sites will only improve over time and
the + helps reflect that it's a delta.

**Comment by april** [comment]
Fixed in 5efbf14, etc.

**Pull request closed without merging**